Rakefile

```diff
@@ -6,7 +6,11 @@ require "ruby_memcheck"
 require "bundler/gem_tasks"
 require "yard"
 
-RubyMemcheck.config(binary_name: "geos_c_impl")
+RubyMemcheck.config(
+  binary_name: "geos_c_impl",
+  # valgrind_generate_suppressions: true,
+  valgrind_suppressions_dir: "test/valgrind/suppressions"
+)
```

Comment:
So `RubyMemcheck` adds a `test:valgrind` task to our list of Rake commands and we just configure it here?

Reply:
Exactly